cap-template.md

```diff
@@ -14,6 +17,32 @@ Protocol version: TBD
 "If you can't explain it simply, you don't understand it well enough." Please provide a simplified
 and layman-accessible explanation of the CAP.
 
+## Working Group
+
+This section describes the composition of the working group.
+
+The following sections are examples of typical changes, and working group composition.
+
+### Semantic protocol changes
+
+For changes that make semantic changes to the protocol, like the addition or change of operations.
```

Comment:
Nitpick: grammar, this is a sentence fragment. I'm not really sure what idea you were trying to express, so I can't reliably suggest a correction.